### PR TITLE
c34 translation

### PR DESCRIPTION
**Pull request by ScrewThisNoise**

### CODE REVIEW

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/105/translation.txt [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
+「……えーと……ちょっと恥ずかしいけど……俺、かな？」=...Erhm... This sounds a bit embarrassing coming from my mouth but... Is it, me?
+「Ｙｅｓ♡　アノネ……帰国する飛行機デ、Ｍａｍａがワタシに言ったノ。『ニッポン語が上手くなりたいなら、恋をするのが一番』ッテ」="Yes♡ You know... When we were on the plane back to Japan, my Mama said to me: If you want to get good at Japanese, fall in love with someone."""
+「恋は、好きなヒトとココロを合わせたい気持ちデ、言葉は、ココロを合わせる恋のＣｕｐｉｄダッテ……Ｍａｍａの言う通りだったヨ♡」="Love is the feeling of wanting to be one of heart with the person you like, and words are the cupids of love that match those hearts"... It was just as Mama said ♡""
+「[H名]のママも、そうだったのかな」=I wonder if your Mama was the same way, [H名].
+「ソウダヨ♡　ワタシが恋をする『オトシゴロ』ナノニ、今でもＰａｐａと毎日Ｌｏｖｅ　ｌｏｖｅダモン♡　コッチがハズカシ～ってなるノ♡」=She was♡ Even after I got to the age of loving someone myself, she still spends all day being lovey-dovey with Papa♡ I feel secondhand embarrassment everytime♡
+「デモ……Ｍａｍａが『今でも、パパに恋してる』って言うと、アナタのコト、思うノ……ウウン、想う……」="But... everytime Mama says Even now, I love Papa", I think of you... No, I 'think' of you... ""
 //「英語は思考の"思う"モ、想像の"想う"モ、ｔｈｉｎｋダケド、アナタにワタシの恋、伝えるためにニッポン語、勉強したヨ。ワタシの"オモウ"、ドッチ？　分かる？」=
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/105/translation.txt:16]: Did something get missed by the tool when merging?

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/2/translation.txt [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-//「フフ、ワタシが育った国、オンナノコ、ミンナａｇｇｒｅｓｓｉｖｅデシタ。アナタ、ワタシのｆｉｒｓｔ　ｌｏｖｅデース」=
-//（ファーストラブって初恋だよな。ちゃんと俺の人となりを見て好きになってくれたのは嬉しいけど、展開が一段飛ばしじゃないか！？）=
-//「アッ！　イマ、恋人になてほしいと違いマスヨ？　コレカラ、モット仲良ししたいいう意味デスノデ、ヨロシクお願いシマス♡」=
-//「[H]の気持ちはちゃんと伝わったよ。これからもよろしくね」=
-//（とは言ったけど、これからは話すだけでドキドキしちゃいそうだな）=
+﻿「この国のコト、スコシ分かてキマシタ。ケド、アナタのコト、トテモよく分かりマシタ」=I feel like I understand a bit what Japan is about now. But, I understand you a whole lot more.
+「え？」=Eh?
+「ワタシ、アナタのコト、好きデス♡　オトコノコ、トシテ、愛、感じマス♡」=I love you♡ I feel love for you, as a woman♡
+「えええっ！？　な、なんで！？」=EEEEEEH!? W-Why!?
+「ナンデ？　Ｏｈ，ｒｅａｓｏｎデスネ？　ニッポン語出来ないワタシに、イッパイ親切くれマシタデスカラ」="Why?" Oh, you're asking about the reason? It's because you showed me so much kindness, a girl who can't into Japanese.""
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/2/translation.txt:5]: It's odd when existing English words don't exist in the translation, even when it's broken English. Also this line has funky quoting.

Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/2/translation.txt:5]: Translator got leave to take some creative freedoms in this personality, as a straight translation would be really awkward, thus some weird sentences, but a straight translation is even more awkward

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/203/translation.txt [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-//「嘘ついタラ、ｔｈｏｕｓａｎｄ　ｎｅｅｄｌｅ～♡」=
-//「サウザンド・ニードルって、なんか技名みたいだね！？」=
+﻿「ア～楽しかったネ♡　次は何にスル？　Ｒｏｌｌｅｒ　ｃｏａｓｔｅｒ……ア、ニッポン語では『ジェットコースター』だったネ♡　もう一回、乗る？」="Ah~ That was fun, wasn't it?♡ What should we ride next? Roller coaster... Ah, it's called a Jet Coaster" in Japanese, isn't it?♡ Wanna go on it one more time?""
+「[H名]、そろそろ夕方だから、考えて乗らないと、すぐ帰る時間になっちゃうよ」=[H名], It's almost evening so, if we don't plan the next little bit out, we'll have to go back home in no time.
+「Ｏｈ……もうそんな時間……ン～、ドウスル？　思ったヨリ、遅い時間だったネ……」=Oh... It's already this late, huh... Nn~, What should we do? It's way later than we expected, isn't it...
+「じゃあ、最後に観覧車に乗ろうか」=Then, let's make the ferris wheel our last ride for the day.
+「Ｎｉｃｅ　ｉｄｅａ♡　今の時間ナラ、ｒｏｍａｎｔｉｃな時間を過ごせそうダネ♡　Ｇｉａｎｔ　ｗｈｅｅｌ、乗りタイ♡」=Nice idea♡ It'd be super romantic at this time♡ I want to ride the giant wheel♡
+「よし、じゃあ、乗ろうか」=Alright, then, let's get on it.
+「んー、今日は暗くなる前に帰ろうか。お楽しみは次に遊園地きたときにとっておこうよ」=Nnn, maybe we should go home before it gets dark. Let's save the fun for the next time we come to the park.
+「フフ♡　『オタノシミ』イイネ♡　ジャア、ニッポン風のヤクソク、シテ♡」="Fufu♡ Save the fun", sounds great♡ Then, promise me that in the Japanese way♡""
```
[the remark this answers is not in the document]
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/203/translation.txt]: should be fixed

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/105/translation.txt (hunk 1 above)
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/105/translation.txt]: Seems fixed

**Review by GeBo1 — APPROVED** [review]

**Review by ManlyMarco — APPROVED** [review]

**Pull request merged**